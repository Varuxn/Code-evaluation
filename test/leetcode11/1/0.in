96 294
6 8 983
31 69 597
50 57 439
25 66 712
54 1 671
28 7 49
56 18 786
68 81 643
57 70 265
66 92 159
11 54 447
43 28 275
7 23 111
45 47 110
62 20 872
16 59 679
87 57 63
78 95 356
4 54 431
48 77 79
38 68 858
34 7 600
38 14 363
9 64 660
77 91 815
71 83 580
39 40 325
22 18 211
2 59 865
45 13 92
1 2 977
31 73 15
3 1 915
14 72 388
50 81 183
33 15 577
28 75 690
9 74 420
3 10 544
26 31 480
5 12 774
47 66 856
45 31 574
54 33 156
62 54 820
76 55 750
87 47 775
17 84 30
47 57 879
21 69 15
19 50 812
25 60 999
16 44 57
92 80 923
6 87 60
32 40 644
13 30 877
79 46 196
45 87 420
74 56 984
60 51 25
21 85 445
17 29 698
87 13 372
16 14 524
15 88 907
37 92 712
11 14 866
34 40 652
0 8 83
2 3 40
83 22 490
12 21 524
0 37 351
30 25 878
10 59 73
42 55 226
86 8 271
40 42 79
81 28 990
29 32 538
14 92 623
22 32 246
0 10 83
36 48 917
57 92 976
35 25 49
84 61 764
12 16 213
93 15 878
61 56 701
22 12 798
17 43 380
29 80 151
58 62 663
73 84 732
45 9 63
87 83 166
22 26 252
70 20 511
23 74 834
12 19 462
1 16 714
47 95 672
92 49 57
30 5 498
87 79 674
42 39 188
0 60 864
91 22 149
27 60 850
2 5 421
30 38 668
32 49 438
79 2 581
86 41 758
74 89 194
9 15 377
4 17 668
8 39 364
63 5 553
2 43 860
54 94 984
4 53 188
1 91 921
2 4 336
45 16 810
60 64 791
18 61 18
24 5 67
24 82 833
22 42 47
9 24 54
34 46 765
4 85 418
70 72 18
77 68 572
48 63 469
14 76 403
36 81 782
86 15 659
86 85 261
46 6 610
6 26 975
42 24 929
6 51 195
2 7 721
45 80 313
77 73 855
73 63 381
7 9 412
82 83 455
63 1 209
29 87 606
24 41 48
41 50 658
42 73 967
43 20 704
53 79 512
6 92 226
12 13 432
39 76 84
7 33 306
25 46 909
19 63 384
68 79 528
16 67 286
52 67 330
13 5 932
69 26 413
37 41 478
48 56 656
24 50 274
7 25 706
5 22 217
36 45 688
10 62 394
84 79 190
42 14 664
0 26 22
62 4 505
13 22 823
8 16 886
28 85 109
88 70 866
91 26 150
74 11 274
9 0 934
53 28 399
46 92 689
29 39 84
55 94 958
39 53 96
44 26 208
9 20 959
89 15 349
43 30 770
51 56 782
83 4 729
38 75 699
49 62 159
77 12 516
0 68 248
88 89 883
2 65 376
77 86 685
75 69 575
57 60 690
74 41 42
75 86 877
40 83 952
61 73 422
7 36 43
24 69 774
35 9 397
1 44 802
68 41 725
36 72 736
7 44 820
17 6 208
32 35 744
52 56 806
15 62 317
18 21 85
38 54 568
57 75 970
52 5 750
1 6 253
67 76 258
77 78 15
81 56 739
19 38 516
17 18 758
76 45 689
14 34 674
24 52 47
18 54 501
1 13 988
76 86 806
65 93 501
94 33 454
27 45 928
92 71 897
9 27 628
6 28 405
11 29 582
65 46 205
33 26 431
42 72 923
38 61 588
68 74 978
48 88 306
87 38 757
27 57 991
14 70 646
34 64 345
12 45 627
26 29 985
28 56 565
81 39 618
0 1 260
24 89 96
24 71 238
40 80 891
66 55 213
89 71 674
19 33 146
66 27 982
92 39 847
14 51 53
26 95 754
79 85 440
60 33 329
60 81 334
4 58 628
28 62 201
88 46 638
55 14 181
34 92 70
50 56 124
8 11 101
16 68 438
32 90 123
20 41 28
27 5 918
94 11 564
56 42 1000
27 12 949
54 67 821
10 45 719
51 75 404
80 67 502
29 47 750
36 74 58
