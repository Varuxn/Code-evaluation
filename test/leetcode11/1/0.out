5 8 9 11 12 13 16 17 19 22 27 29 31 34 43 47 49 52 54 57 60 69 70 73 75 78 82 83 86 88 93 96 97 104 105 107 109 123 125 128 129 131 132 135 137 141 145 149 151 152 154 161 162 172 174 177 179 183 185 190 192 193 200 202 208 212 214 219 222 223 227 228 229 235 239 240 243 246 251 255 261 262 264 266 269 274 275 277 278 279 280 282 283 290 293 

